pkg/checks/external/main.go

```diff
@@ -1050,12 +1050,12 @@ func (ext *Checker) createPod() (*apiv1.Pod, error) {
 	// enforce various labels and annotations on all checker pods created
 	ext.addKuberhealthyLabels(p)
 
-	// Get ownerRefernece for the kuberhealthy pod
-	ownerRef, err := util.GetOwnerRef(ext.KubeClient, ext.Namespace)
+	// Get ownerReference for the kuberhealthy pod
+	ownerRef, err := util.GetOwnerRef(ext.KubeClient, "kuberhealthy")
```

Comment:
shouldn't "kuberhealthy" be ext.Namespace? what about checks outside of the kuberhealthy namespace